arb_os/gasAccounting.mini

```diff
@@ -228,18 +228,18 @@ view throw func initCongestionState(isInit: bool) -> CongestionState {
                     gasPoolMax * x / 100
                 }
             } else {
-                gasPoolMax / 2
+                const::Default_Param_GasPoolFullnessTargetPercent
```

Comment:
This needs to be multiplied by `gasPoolMax` then divided by 100.

Reply:
Nice catch, fixed